### PR TITLE
[NOTAM-37] Add Client Airport Codes

### PR DESCRIPTION
**Pull request by jamesf31**

### CODE REVIEW

**Comment by tanminivan** [comment]
Where did the docstring hurt you?

**Review by swagatron300 — APPROVED** [review]

**Review by Treeeeeeee — APPROVED** [review]

**Review by KamidzuKu — APPROVED** [review]

**Comment by tanminivan** [comment]
Honestly I'd leave the docstrings alone but I would not hold up this PR because of it.

File: backend/tests/test_GeoUtilities.py [hunk 1]
```diff
@@ -7,43 +7,50 @@
...
         self.assertEqual(round(coordinate.latitude, 2), 35.39)
         self.assertEqual(round(coordinate.longitude, 2), -97.60)
 
     def test_get_icao_coordinate(self):
-        """Test getting a coordinate for a valid ICAO code. Rounded to two decimal places."""
+        #Test getting a coordinate for a valid ICAO code. Rounded to two decimal places.
         coordinate = self.geo_utilities.geo_resolve('KDFW')
         self.assertIsInstance(coordinate, Coordinate)
-        self.assertEqual(round(coordinate.latitude, 2), 32.9)
+        self.assertEqual(round(coordinate.latitude, 2), 34.458625)
```
Comment (tanminivan) [inline backend/tests/test_GeoUtilities.py:30]: Was this value just completely wrong?

Comment (jamesf31) [reply backend/tests/test_GeoUtilities.py:30]: Nope, I erroneously changed it. Fixing that, then I'll rebase and merge.

**Pull request merged**